test/listen.test.js

```diff
@@ -351,3 +351,23 @@ test('listen logs the port as info', t => {
       t.ok(/http:\/\//.test(msgs[0]))
     })
 })
+
+test('listen when firstArg is string(pipe) and without backlog', t => {
+  t.plan(1)
+  const fastify = Fastify()
+  t.tearDown(fastify.close.bind(fastify))
+  fastify.listen('\\\\.\\pipe\\testPipe')
```

Comment:
can you please convert this to `await fastify.listen('\\\\.\\pipe\\testPipe')`

Reply:
Completed. Thanks for the suggestion !